ui/src/templates/components/CommunityTemplateImportOverlay.tsx

```diff
@@ -96,6 +97,7 @@ class UnconnectedTemplateImportOverlay extends PureComponent<Props> {
       return summary
     } catch (err) {
       this.props.notify(communityTemplateInstallFailed(err.message))
+      reportError(err, {name: 'The community template install failed'})
```

Comment:
this is happening in the review templates section, let's change the name to make it a specific:
```suggestion
      reportError(err, {name: 'The community template fetch for preview failed'})
```